### DISCUSSION TITLE
Presenting: `saleor-next-starter` 🏎️

### DISCUSSION THREAD

**Discussion by peelar** [saleor/saleor#12240] [Announcements]
Hi folks 👋 

We are expanding our front-end boilerplate library with [`saleor-next-starter`](https://github.com/saleor/saleor-next-starter). It's an opinionated Next.js starter that focuses on solving the core challenges of integrating with Saleor, like:
- GraphQL setup (with codegen)
- Authentication
- Server-side calls

The difference between [`saleor-next-starter`](https://github.com/saleor/saleor-next-starter) and [`react-storefront`](https://github.com/saleor/react-storefront) is that the latter is a complete demo repository you can use to explore the features of Saleor. _Starter_ is meant to be a place to start building your own Saleor storefront.

This is just a first step in building a variety of starters for different frameworks. Let us know in the comments what frameworks you want to see next (Vue? Svelte? Remix?) and what fundamental Saleor feature you want us to cover the most.

**Comment by flachdaniel** [comment]
Nice repo,
Next 13's app directory approach would be appreciated. Its in beta but as soon as it goes out to live, this will be the preferred way of writing next applications. A guide on how to use the new 13.2 api fetching with saleor's graphql for SSR would be a great way to start building your own storefront for the future.

**Reply by timuric** [reply]
@flachdaniel we are exploring a new demo store with app approach already, but Vercel team is not recommending to use it in production yet, so it is in research phase still.

**Comment by c00lcoder** [comment]
A starter that uses nextjs, next-auth and configured for digital products including secure download urls for purchases would be huge

**Comment by zanami** [comment]
Please implement a simple integrated cart/order/checkout process so this project could be an alternative to react-storefront which relies on  Checkout App.

**Comment by apeconstant** [comment]
There isn't any resource available for Svelte. saleor-svelte-starter would be much appreciated.

**Reply by bgronek** [reply]
I'll second that. I'd love to see a Svelte starter!

**Reply by j2l** [reply]
I'd love this too!

**Reply by timuric** [reply]
Unfortunately we don't have svelte users in our team, but contributions are welcome!
